src/test/skript/tests/syntaxes/conditions/CondIsPlain.sk

```diff
@@ -0,0 +1,4 @@
+test "itemtype is plain":
```

Comment:
should include testing for items that aren't plain

Reply:
Every item not done with the plain expression are not plain which is why I found it dumb that a simple `emerald` call was considered not plain some how.